rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -30,7 +31,20 @@ def test_executable():
         else:
             exit_handler = ignore_exit_handler
             exit_on_match = False
-        handler = create_handler(name, launch_descriptor, output_file, exit_on_match=exit_on_match)
+
+        filtered_prefixes = [
+            b'pid', b'rc',
```

Comment:
Currently the default filtering for launch includes filtering RTI licence output at https://github.com/ros2/launch/blob/master/launch_testing/launch_testing/__init__.py#L33
I'm trying to work towards making this (`b'pid', b'rc'`) the default for filtered_prefixes with https://github.com/ros2/launch/pull/24 then this line won't be necessary